west/settings.py

```diff
@@ -41,8 +43,20 @@ def set(self, **kwargs):
         vars(self).update(kwargs)
 
 
+def log_level(verbosity):
+    """
+    increasing verbosity levels are 0, 1, 2, 3, ...
+    corresponding log levels 40, 30, 20, 10
+    """
+    return 40 - 10 * min(verbosity, 3)
```

Comment:
but who knows what are the value corresponding to DEBUG, INFO, ... ?

Reply:
done